src/format/svg/SVGData.hx

```diff
@@ -35,6 +35,7 @@ class SVGData extends Group {
 	private static var mScaleMatch = ~/scale\((.*)\)/;
 	private static var mMatrixMatch = ~/matrix\((.*)[, ](.*)[, ](.*)[, ](.*)[, ](.*)[, ](.*)\)/;
 	private static var mURLMatch = ~/url\(#(.*)\)/;
+	private static var mRGBMatch = ~/rgb\s*\(\s*(\d+)\s*(%)?\s*,\s*(\d+)\s*(%)?\s*,\s*(\d+)\s*(%)?\s*\)/;

```

Comment:
I played around with this on [Rubular](http://rubular.com) and I see very strange results.

If I enter `rgb(255, 128, 0)`, I get six groups:
- 255
- Blank
- 128
- Blank
- 0
- Blank

I can also enter `rgb(255, 17%, 0)`, which is strange, but I'm not sure if it's worth the effort to throw an error if the user does something silly like this.

What do you think?